chapter_5_practic/task9.cpp

```diff
@@ -0,0 +1,52 @@
+//Repeat Exercise 8, but instead of two int variables, have the swap() function interchange two struct time values(see Exercise 6)
+
+#include <iostream>
+
+struct user_time
+{
+	int hours = 0;
+	int minutes = 0;
+	int seconds = 0;
+};
+
+user_time time1;
```

Comment:
why do you create a global variable?

Reply:
don't understand why)
https://github.com/Dmitriy-Romanchuk/master/pull/5/commits/d51837670d39919f84eaf71c4f9253626c561454